doc/source/template_docker_nextflow.config

```diff
@@ -18,11 +17,17 @@ process {
     // Specify memory and number of threads for processes submitted to the cluster 
     // It is advised to provide higher computational power to wgdParalogs and wgdOrthologs 
     withName: 'wgdParalogs' {
-        clusterOptions = ''  // e.g. '-pe serial 10 -l h_vmem=2G'
+        cpus = ''            // e.g. '8'
+        penv = ''            // e.g. 'serial'
+        memory = ''          // e.g. '16GB'
+        clusterOptions = ''  // e.g. '-l h_vmem=2G'
     }
     
     withName: 'wgdOrthologs' {
-        clusterOptions = ''  // e.g. '-pe serial 10 -l h_vmem=2G'
+        cpus = ''            // e.g. '8'
+        penv = ''            // e.g. 'serial'
+        memory = ''          // e.g. '16GB'
```

Comment:
Do cpus and memory values need to be in quotes or does it matter?

Reply:
Wouldn't it make sense to also add at least the `memory` setting also for the other processes below and not just `clusterOptions`?

Reply:
Agreed!